.github/workflows/flood-smartnet.yml

```diff
@@ -0,0 +1,59 @@
+name: flood-smartnet
+
+on:
+  pull_request:
+    paths-ignore:
+      - '*.md'
+      - '.github/workflows/**'
+    branches:
+      - smartnet
+  push:
+    paths-ignore:
+      - '*.md'
+      - '.github/workflows/**'
+    branches:
+      - smartnet
+
+concurrency:
+  group: ${{ github.ref }}-${{ github.workflow }}
+  cancel-in-progress: true
+
+jobs:
+  build-node:
+    name: Build node and runtime artifacts (PR version)
+    uses: Cardinal-Cryptography/aleph-node/.github/workflows/build-node-and-runtime.yml@smartnet
+    with:
+      rust-toolchain-version: 'nightly-2021-10-24'
+
+  test-contracts-flooding:
+    name: Test contracts flooding
+    runs-on: ubuntu-latest
+    needs: [ build-node ]
+    env:
+      RUST_TOOLCHAIN_VERSION: nightly-2021-10-24
```

Comment:
After this is merged https://github.com/Cardinal-Cryptography/aleph-node/pull/298, we don't need to state `RUST_TOOLCHAIN_VERSION` explicetely?

Reply:
yes